jenkins-build-driver/src/test/java/org/jboss/pnc/jenkinsbuilddriver/test/JenkinsDriverRemoteTest.java

```diff
@@ -143,12 +142,12 @@ public RepositorySession getRepositorySession() {
             
             @Override
             public String getJenkinsUrl() {
-                return System.getProperty("PNC_JENKINS_URL");
+                return System.getenv("PNC_JENKINS_URL") + ":" + getJenkinsPort();
```

Comment:
Will developers be able to run this test without setting the System env variable ?